tests/synthesizer/conftest.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from aioresponses import aioresponses, CallbackResult
+from vocode.streaming.models.audio_encoding import AudioEncoding
+from vocode.streaming.models.synthesizer import ElevenLabsSynthesizerConfig
+import re
+from vocode.streaming.synthesizer.eleven_labs_synthesizer import (
+    ElevenLabsSynthesizer,
+    ELEVEN_LABS_BASE_URL,
+)
+import re
+from tests.streaming.data.loader import get_audio_path
+
+DEFAULT_PARAMS = {'sampling_rate': 16000, 'audio_encoding': AudioEncoding.LINEAR16}
+
+def create_request_handler(optimize_streaming_latency=False):
+    def request_handler(url, headers, **kwargs):
+        if optimize_streaming_latency and "optimize_streaming_latency=1" not in str(url):
```

Comment:
probably not good to include `1` as a magic number here - can we use a regex that checks that `optimize_streaming_latency=\d` is in the URL?